Extension/src/LanguageServer/ui.ts

```diff
@@ -626,16 +596,6 @@ export class LanguageStatusUI {
         this.showConfigurationPrompt(ConfigurationPriority.IncludePath, prompt, onSkip);
     }
 
-    public showConfigureCompileCommandsMessage(prompt: () => Promise<boolean>, onSkip: () => void): void {
-        setTimeout(() => {
-            this.showConfigurationPrompt(ConfigurationPriority.CompileCommands, prompt, onSkip);
-        }, 5000);
-    }
-
-    public showConfigureCustomProviderMessage(prompt: () => Promise<boolean>, onSkip: () => void): void {
-        this.showConfigurationPrompt(ConfigurationPriority.CustomProvider, prompt, onSkip);
-    }
-
     private showConfigurationPrompt(priority: ConfigurationPriority, prompt: () => Thenable<boolean>, onSkip: () => void): void {
```

Comment:
`showConfigurationPrompt` and `showConfigureIncludePathMessage` should be deleted

Reply:
`showConfigureIncludePathMessage` is called from `message.endsWith("Unresolved Headers")` case. I'm assuming "Unresolved Headers" is handled differently now. I can't remember what that changed was.